optaplanner-test/src/test/java/org/optaplanner/test/api/score/stream/testdata/domain/TestdataValue.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.test.api.score.stream.testdata.domain;
+
+public class TestdataValue extends TestdataObject {
```

Comment:
Why not add a "test-jar" dependency on optaplanner-core and reuse the TestdataValue from the test classes of optaplanner-core?